test/patterns/method_chaining/test_method_chaining.py

```diff
@@ -32,63 +32,63 @@ class TestMethodChain(TestCase):
 
     def test_method_chain(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'MethodChain.java'))
-        self.assertEqual(len(lines), 1)
+        self.assertEqual(lines, [21])
 
     def test_empty_method_chain(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'EmptyMethodChain.java'))
-        self.assertEqual(len(lines), 1)
+        self.assertEqual(lines, [21])
 
     def test_chain_with_new_object(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'MethodChainNewObjectMethods.java'))
-        self.assertEqual(len(lines), 1)
+        self.assertEqual(lines, [34])
 
     def test_method_chain_in_different_methods(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'MethodChainInDifferentMethods.java'))
-        self.assertEqual(len(lines), 2)
+        self.assertEqual(lines, [22, 34])
 
     def test_chain_in_nested_class(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'MethodChainNestedClass.java'))
-        self.assertEqual(len(lines), 1)
+        self.assertEqual(lines, [19])
 
     def test_chain_in_anonymous_class(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'MethodChainAnonymousClass.java'))
-        self.assertEqual(len(lines), 1)
+        self.assertEqual(lines, [29])
 
     def test_chain_in_anonymous_class_empty(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'MethodChainAnonymousClassEmpty.java'))
-        self.assertEqual(len(lines), 0)
+        self.assertEqual(lines, [])
 
     def test_several_chains(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'MethodChainSeveral.java'))
-        self.assertEqual(len(lines), 3)
+        self.assertEqual(lines, [13, 34, 48])
 
     def test_chain_without_object_creating(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'WithoutObjectCreating.java'))
-        self.assertEqual(len(lines), 1)
+        self.assertEqual(lines, [14])
 
     def test_nested_chain_with_this(self):
         lines = self.method_chain_finder.value(Path(self.dir_path, 'NestedChainWIthThis.java'))
-        self.assertEqual(len(lines), 2)
```

Comment:
Failed test case:

obj.ohhhh().set(this.govno("Zh").zhopa(whaaat).concat(whaaat)).opopo(whaaat);

1) this.govno("Zh").zhopa(whaaat).concat(whaaat)
2) obj.ohhhh().set().opopo(whaaat);

Reply:
Both lines are found now.